figures/monolith-id-sequence.mmd

```diff
@@ -0,0 +1,13 @@
+sequenceDiagram
+
+    actor Jeff
+    Participant OTT
+    participant Monolith
+    participant Balancer
+
+    Note over OTT: Load Balancing Enabled
+    Jeff->>OTT: Goes to OTT
+    OTT->>Monolith: Client connects to Monolith
```

Comment:
With the balancer, clients never connect to the monolith directly.

Reply:
Do you just want to get rid of the Client part entirely?